app/src/main/java/com/jerboa/MainActivity.kt

```diff
@@ -446,72 +518,6 @@ class MainActivity : AppCompatActivity() {
-                            appState::navigateUp,
-                        ) {
-                            PostActivity(
-                                id = Either.Left(args.id),
-                                accountViewModel = accountViewModel,
-                                appState = appState,
-                                showCollapsedCommentContent = appSettings.showCollapsedCommentContent,
-                                showActionBarByDefault = appSettings.showCommentActionBarByDefault,
-                                showVotingArrowsInListView = appSettings.showVotingArrowsInListView,
-                                showParentCommentNavigationButtons = appSettings.showParentCommentNavigationButtons,
-                                navigateParentCommentsWithVolumeButtons = appSettings.navigateParentCommentsWithVolumeButtons,
-                                siteViewModel = siteViewModel,
-                                useCustomTabs = appSettings.useCustomTabs,
-                                usePrivateTabs = appSettings.usePrivateTabs,
-                                blurNSFW = appSettings.blurNSFW,
-                                showPostLinkPreview = appSettings.showPostLinkPreviews,
-                                postActionbarMode = appSettings.postActionbarMode,
-                            )
-                        }
-                    }
-
-                    composable(
-                        route = Route.COMMENT,
-                        deepLinks = DEFAULT_LEMMY_INSTANCES.map { instance ->
-                            navDeepLink { uriPattern = "$instance/comment/{${Route.CommentArgs.ID}}" }
-                        },
-                        arguments = listOf(
-                            navArgument(Route.CommentArgs.ID) {
-                                type = Route.CommentArgs.ID_TYPE
-                            },
-                        ),
-                    ) {
-                        val args = Route.CommentArgs(it)
-                        PostActivity(
-                            id = Either.Right(args.id),
-                            accountViewModel = accountViewModel,
-                            appState = appState,
-                            useCustomTabs = appSettings.useCustomTabs,
-                            usePrivateTabs = appSettings.usePrivateTabs,
-                            showCollapsedCommentContent = appSettings.showCollapsedCommentContent,
-                            showActionBarByDefault = appSettings.showCommentActionBarByDefault,
-                            showVotingArrowsInListView = appSettings.showVotingArrowsInListView,
-                            showParentCommentNavigationButtons = appSettings.showParentCommentNavigationButtons,
-                            navigateParentCommentsWithVolumeButtons = appSettings.navigateParentCommentsWithVolumeButtons,
-                            siteViewModel = siteViewModel,
```

Comment:
the toComment crashes because you removed this composable.

This is the one that makes it possible